.travis.yml

```diff
@@ -31,6 +31,7 @@ install:
   - pip install python-coveralls
   - pip install easybuild
   - pip install -r docs/requirements.txt
+  - pip install -r requirements.txt
```

Comment:
i see we now have two `requirements.txt` one in the root and the one originally under `docs/requirements.txt`.

I think this will cause confusion for users and us. I know the requirements.txt has a whole bunch of deps that are need to be refined. Is it necessary for having this installed as part of travis.

Reply:
I think now that we have the python package, we don't need the requirements.txt in the root. The docs/requrements.txt can be left to help developers that want to install document building dependencies. I'll remove this.

Reply:
Removed.